src/Uno.UI.XamlHost.Skia.Wpf/UnoXamlHostBase.Focus.cs

```diff
@@ -111,6 +111,7 @@ protected override void OnGotFocus(RoutedEventArgs e)
         //    }
         //}
 
+#if false
```

Comment:
I didn't delete this because there is a commented usage above:

https://github.com/unoplatform/uno/blob/bc1d0297d052a62e4df6faa087b61e1353b86bcd/src/Uno.UI.XamlHost.Skia.Wpf/UnoXamlHostBase.Focus.cs#L92

So in case the code will be uncommented at some point, I wanted to make it easy to find this method.